app/bundles/EmailBundle/Helper/MessageHelper.php

```diff
@@ -2144,4 +2167,63 @@ public static function parseAddressList($addresses)
 
         return $results;
     }
+    
+    public function processReplyMail($mailId, $refid)
+    {
+        $this->logger->debug('Processing reply mail.');
+        $sendFolderName = '';
+        foreach($this->mailbox->getListingFolders() as $folderNames) {
+            if(strpos(strtolower($folderNames), 'sent') !== false) {
+                $sendFolderName = $folderNames;
+                break;
+            }
+        }
+        $this->mailbox->switchFolder($sendFolderName);
+        $mailIds = $this->mailbox->searchMailbox('TO '.$mailId);
+        $mails   = $this->mailbox->getMailsInfo($mailIds);
+        foreach ($mails as $mail) {
+            if ($mail->message_id == $refid) {
+                $this->checkMail($mail->uid);
+            } 
+        }
+    }
+    
+    public function checkMail($mailUid)
```

Comment:
Can we add some code to attempt filtering out auto responder messages? Not sure if we just need to hard code a list of most common auto responders or if there is a header email client set that we can latch onto.